mbq/metrics/contrib/django/middleware/timing.py

```diff
@@ -0,0 +1,50 @@
+from time import time
+
+from django.conf import settings
+try:
+    from django.utils.deprecation import MiddlewareMixin as MiddlewareDeprecationMixin
+except ImportError:
+    MiddlewareDeprecationMixin = object
+
+import mbq.metrics as metrics
+
+DEFAULT_SETTINGS = {
+    'EXCLUDED_PATHS': []
+}
+
+SETTINGS = DEFAULT_SETTINGS.copy()
+SETTINGS.update(
+    getattr(settings, 'MBQ_METRICS', {})
+)
+
+
+class TimingMiddleware(MiddlewareDeprecationMixin):
+
+    def __init__(self, get_response=None):
+        self.get_response = get_response
+
+    def process_request(self, request):
+        if request.path in SETTINGS['EXCLUDED_PATHS']:
```

Comment:
Seems error prone. If `/api/v1/offices` was excluded and the actual path was `/api/v1/offices/`, it'd be hard to debug and you'd end up being at the mercy of Django's request parsing and the APPEND_SLASH setting.

Reply:
Alternatively, maybe we can preprocess the excluded paths to strip leading and trailing slashes and then just `request.path.strip('/')`

Reply:
it's meant to be extremely hacky. We should make this more robust if it is a useful feature! 

I incorporated your suggestions though!

Reply:
✅